pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/util/gui/unit/UnitFormat.kt

```diff
@@ -10,6 +10,20 @@ import java.math.RoundingMode
 import java.time.Duration
 import java.util.EnumSet
 
+/**
+ * Handles formatting of a specific unit. Call [format] to format a value using this unit.
+ *
+ * @param name The English name of the unit (for example 'kilograms')
+ * @param singular A component representing the long singular form of this unit (kilogram, metre, litre, etc)
+ * @param plural A component representing the long plural form of this unit (kilograms, metres, litres, etc)
```

Comment:
```suggestion
 * @param singular A component representing the long singular form of this unit (kilogram, metr, litr, etc)
 * @param plural A component representing the long plural form of this unit (kilograms, meters, liters, etc)
```
Our lang files uses the Superior English